rust/otap-dataflow/crates/otap/src/batch_processor.rs

```diff
@@ -152,22 +221,87 @@ const fn default_outbound_request_limit() -> NonZeroUsize {
     NonZeroUsize::new(512).expect("ok") // default_min_size/16
 }
 
+const fn default_batching_format() -> BatchingFormat {
+    BatchingFormat::Otap
```

Comment:
I wonder if this should be `Preserve`? 

If batching OTLP Bytes is more efficient than converting to OTAP and batching arrow records, then having this as `Preserve` would mean users doing `OTLP Receiver -> Batch Processor -> OTLP Exporter` would not have to opt-in to the most efficient pipeline